frontend/packages/local-storage-operator-plugin/src/models.ts

```diff
@@ -0,0 +1,27 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const AutoDetectVolumeModel: K8sKind = {
+  label: 'Volume',
+  labelPlural: 'Volumes',
+  apiVersion: 'v1alpha1',
+  apiGroup: 'local.storage.openshift.io',
+  plural: 'volumes',
+  abbr: 'ADV',
+  namespaced: true,
+  kind: 'AutoDetectVolume',
+  id: 'autodetectvolume',
+  crd: true,
+};
+
+export const LocalVolumeModel: K8sKind = {
+  label: 'Local Volume',
+  labelPlural: 'Local Volumes',
+  apiVersion: 'v1',
+  apiGroup: 'local.storage.openshift.io',
+  plural: 'localvolumes',
+  abbr: 'LV',
+  namespaced: true,
+  kind: 'LocalVolume',
+  id: 'localvolume',
+  crd: true,
+};
```

Comment:
@ashishranjan738 Can you please review the models as well for `AutoDetectVolumeModel` and `LocalVolumeModel`

Reply:
@gnehapk There is nothing called `AutoDetectVolume` CR any more it has been renamed to `LocalVolumeGroup`. You can find the finalized designs for it here https://github.com/openshift/enhancements/blob/master/enhancements/local-storage/automatic-detection.md.

Reply:
The `LocalVolume` Model seems to okay